### DISCUSSION TITLE
Why is `onEnter` necessary for animation

### DISCUSSION THREAD

**Discussion by tylersayshi** [mantinedev/mantine#381] [Q&A] [answered]
Hi, I just noticed this line https://github.com/mantinedev/mantine/blob/master/src/mantine-notifications/src/NotificationsProvider/NotificationsProvider.tsx#L89 is necessary for ensuring that the enter animation works correctly for the notifications.

Do you happen to remember why this was needed? It seems at first glance like this does nothing at all.

**Comment by rtivital** [comment, marked as answer]
You can see full explanation here – https://github.com/reactjs/react-transition-group/issues/223
